### ISSUE TITLE
Remove vibrationManager from Sim.

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/joist#794]
While porting Sim.js to Sim.ts, I ran into this field:

```js
      // {VibrationManager|null} - Responsible for managing vibration feedback for a sim. Experimental, and
      // not used frequently. The vibrationManager instance is passed through options so that tappi doesn't have to
      // become a dependency for all sims yet. If this gets more use, this will likely change.
      vibrationManager: null,
````

After converting to .ts, "tappi doesn't have to become a dependency for all sims yet" is not longer true. A the very least, tappi would need to be added to joist/tsconfig.json.  It might also need to be added to build.json.

I asked @jessegreenberg about this on Slack, and he said:
> I think usages of vibrationManager can be removed from Sim now. There are no plans to continue with that right now. I can remove the usage from john-travoltage.

So vibrationManager is going to be removed from Sim.ts.  Since I'm partway through convertion, I'm just going to add `// @ts-ignore` for an vibrationManager code.  Then I'll assign to @jessegreenberg for cleanup.

**Comment by pixelzoom** [comment]
In duplicate https://github.com/phetsims/joist/issues/793, @jessegreenberg said:

> There are no plans in the next year or more to use Web based vibration. The only usage is in john-travoltage. We should remove it from both john-travoltage and Sim.js at this time. This came up as Sim.js is being converted to TypeScript. VibrationManager was only optionally included so that tappi didn't need to be a dependency for all sims, but strong typing would change that.
>
> On hold until Sim TypeScript conversion is complete and changes can be made to Sim.ts.

**Comment by pixelzoom** [comment]
I bailed on Sim.js conversion to .ts in #795, so Sim is still in JavaScript for the time being.

@jessegreenberg over to you. You might want to remove vibrationManager from Sim.js before someone else spends time porting those bits.

**Comment by jessegreenberg** [comment]
tappi/vibrationManager has been removed from Sim.js, closing.
